include/TS3Data.h

```diff
@@ -21,11 +23,16 @@ class TS3Data : public TGRSIDetectorData  {
     std::vector<UShort_t>  fS3_SectorDetector;
     std::vector<UShort_t>  fS3_SectorNumber;
     std::vector<TFragment> fS3_SectorFragment;
+    
+   static bool fIsSet;      //!
```

Comment:
I'm happy to follow the efficient style ... but this is the same in griffindata as s3data
"bool TGriffinData::fIsSet = false;"

Could you highlight where you mean static variables? Its possible I copied something wrong when trying to match other detectors.